src/test/java/codeu/model/store/basic/ActivityStoreTest.java

```diff
@@ -0,0 +1,52 @@
+package codeu.model.store.basic;
+
+import codeu.model.data.Activity;
+import codeu.model.store.persistence.PersistentStorageAgent;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class ActivityStoreTest {
+
+  private ActivityStore activityStore;
+  private PersistentStorageAgent mockPersistentStorageAgent;
+
+  private final Activity ACTIVITY_ONE =
+      new Activity(Activity.Type.LOGGED_IN, UUID.randomUUID(), Instant.ofEpochMilli(1000), UUID.randomUUID());
+
+  @Before
+  public void setup() {
+    mockPersistentStorageAgent = Mockito.mock(PersistentStorageAgent.class);
+    activityStore = ActivityStore.getTestInstance(mockPersistentStorageAgent);
+
+    final List<Activity> activityList = new ArrayList<>();
+    activityList.add(ACTIVITY_ONE);
+    activityStore.setActivities(activityList);
+  }
+
+  @Test
+  public void testAddActivity() {
+    Activity inputActivity =
```

Comment:
Club lines 34 & 37 together.
Then do `Activity resultActivity = activityStore.getAllActivities().get(1);`

This way, the test will have 3 blocks: 1) All inits 2) Behavior being tested 3) Verifications on the behavior